superset-frontend/src/explore/components/controls/MetricControl/AdhocMetricOption.jsx

```diff
@@ -48,7 +48,7 @@ class AdhocMetricOption extends React.PureComponent {
   }
 
   onRemoveMetric(e) {
-    e.stopPropagation();
+    if (e !== undefined) e.stopPropagation();
```

Comment:
Same as `e?.stopPropagation()` ?

Reply:
using this as a defense statement some cases `e` would come back null